lib/views/menu.erb

```diff
@@ -1,5 +1,21 @@
 <h1>Our Menu</h1>
 
-  <% @dishes.each do | dish | %>
-  <p><%= dish.name %>, <%= dish.price %></p>
-  <% end %>
+<h2>Categories</h2>
```

Comment:
We probably don't need to see Categories here, right? From a design perspective, does it make sense?